lib/mix/lib/mix/tasks/help.ex

```diff
@@ -4,11 +4,12 @@ defmodule Mix.Tasks.Help do
   @shortdoc "Prints help information for tasks"
 
   @moduledoc """
-  Lists all tasks or prints the documentation for a given task.
+  Lists all aliases and tasks or prints the documentation for a given task.
 
   ## Arguments
 
-      mix help                  - prints all tasks and their shortdoc
+      mix help                  - prints all aliases, tasks and task shortdoc
+      mix help ALIAS            - prints the definition for the given alias
       mix help TASK             - prints full docs for the given task
       mix help --search PATTERN - prints all tasks that contain PATTERN in the name
```

Comment:
"prints all tasks and aliases that contain PATTERN in the name"?

Reply:
good catch!